packages/monitor-v2/src/monitor-og/MonitorLogger.ts

```diff
@@ -35,17 +49,20 @@ export async function logTransactions(
       new Date(Number(transaction.proposalTime.toString()) * 1000).toUTCString() +
       " in transaction " +
       createEtherscanLinkMarkdown(transaction.tx, params.chainId) +
-      ". Rules: " +
-      tryHexToUtf8String(transaction.rules) +
       ". Explanation: " +
       tryHexToUtf8String(transaction.explanation) +
       ". The proposal can be disputed till " +
       new Date(Number(transaction.challengeWindowEnds.toString()) * 1000).toUTCString() +
       ": " +
       generateOOv3UILink(transaction.tx, transaction.ooEventIndex, params.chainId) +
       ".",
+    rules: tryHexToUtf8String(transaction.rules),
```

Comment:
For too long rules added to mrkdwn part this caused Slack transport to redact some of the text as it was unable to split it into multiple blocks. Thus, rules now are reported in their own property and this also improves readability.

Reply:
Great idea!